crypto/sm3/m_sm3.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2017 Ribose Inc. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include "internal/cryptlib.h"
+
+#ifndef OPENSSL_NO_SM3
+
+#include <openssl/evp.h>
+#include "internal/evp_int.h"
+#include "internal/sm3.h"
```

Comment:
A space between the `#` and `include` for these.

Reply:
Addressed.